src/compile/mark/mark.ts

```diff
@@ -277,6 +277,21 @@ function getMarkGroup(model: UnitModel, opt: {fromPrefix: string} = {fromPrefix:
   const key = encoding.key;
   const sort = getSort(model);
   const interactive = interactiveFlag(model);
+
+  // set pointer cursor for point selections that are not bound
+  if (
+    interactive &&
+    Object.values(model.component.selection).some(
+      (s) =>
+        s.type === 'point' &&
+        !s.bind &&
+        // if on is a pointerover (hover) the pointer makes less sense since the mark is not clickable.
+        (s as any).on !== 'pointerover',
```

Comment:
Is there a type we could use instead of `any` for checking the presence of `on` ?